### DISCUSSION TITLE
Blank rejection

### DISCUSSION THREAD

**Discussion by mine-tech-oficial** [community/community#89801] [GitHub Education]
### Hi everyone,

After selecting my email, school, and posting my student id, I was rejected with the following message:

> Unfortunately, we weren't able to approve your educational discount request for (my username)

There is no rejection reason after that

**Comment by moritakaaz** [comment]
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your discussion was locked and closed because it contained content that violates our Community Code of Conduct, including posting the same comment/discussion in multiple places. [This is original discussion that supersedes this, and we ask that you no longer cross-post.](https://github.com/orgs/community/discussions/52169)

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service)
